src/_util/caller.js

```diff
@@ -23,12 +23,12 @@ const isWaiting = ({ lastCallTimestamp, created }) => {
 }
 
 
-const callerMonthsLapsed = ({ lastCallTimestamp, lastReminderTimestamp }) => {
-    const lastCallDate = DateTime.fromSQL(lastCallTimestamp)
+const callerMonthsLapsed = ({ lastCallTimestamp, lastReminderTimestamp, created }) => {
+    const lastCallDate = lastCallTimestamp !== null ? DateTime.fromSQL(lastCallTimestamp) : DateTime.fromSQL(created)
```

Comment:
Are we explicitly checking for `null` here? If not, we can just make this `lastCallTimestamp ? DateTime...`

Reply:
Oh other idea! This could also be simplified to:
```
const lastCallDate = DateTime.fromSQL(lastCallstamp || created)
```